wiki/Ranking_Criteria/osu!mania/id.md

```diff
@@ -0,0 +1,199 @@
+# Ranking criteria osu!mania
+
+**Ranking Criteria osu!mania** merupakan peraturan dan pedoman yang berlaku untuk pembuatan tingkat kesulitan spesifik osu!mania. Untuk dapat membuat tingkat kesulitan spesifik osu!mania diwajibkan mematuhi kriteria yang tercantum. Meskipun **semua aturan wajib diikuti dalam keadaan apa pun**, namun pedoman dapat diabaikan dalam kasus tertentu. Kasus tertentu ini harus dapat dibenarkan dengan penjelasan yang lengkap tentang mengapa pedoman tersebut diabaikan dan alasan mengapa tidak mengabaikannya akan mengganggu kualitas keseluruhan pembuatan.
+
+Proposal mengenai peraturan serta saran dibahas di [Sub-forum Ranking Criteria](https://osu.ppy.sh/community/forums/87). Setiap aturan yang melalui proses diskusi dan persetujuan komunitas dicantumkan di sini sebagaimana telah disepakati pada masing-masing utas diskusi.
+
+## Glosarium
+
+### Penamaan tingkat kesulitan
+
+*Artikel utama: [Penamaan Tingkat Kesulitan](/wiki/Ranking_Criteria/Difficulty_Naming)*
+
+- ![](/wiki/shared/diff/easy-m.png) Easy
+- ![](/wiki/shared/diff/normal-m.png) Normal
+- ![](/wiki/shared/diff/hard-m.png) Hard
+- ![](/wiki/shared/diff/insane-m.png) Insane
+- ![](/wiki/shared/diff/expert-m.png) Expert
+
+### Istilah umum
+
+- **BPM**: Tempo lagu yang ditentukan berdasarkan ketukan per menit.
+- **Column**: Garis vertikal dalam area permainan di mana sebuah note ditempatkan. Editor menggunakan `0|1|2|3|4|5|6|7|8` untuk label timestamp.
```

Comment:
```suggestion
- **Column**: Garis vertikal dalam area permainan di mana sebuah note ditempatkan. Editor menggunakan `0|1|2|3|4|5|6|7|8|9` untuk label timestamp.
```

Reply:
mmmm i'm sure there's no "9" in master, so i'll keep this one